docs/program/apis/ml_training_client.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Work with ML Training Jobs with Viam's ML Training API"
+linkTitle: "ML Training Client"
+weight: 10
+type: "docs"
+description: "Use the ML training client API to manage ML training jobs taking place in Viam's cloud app."
+tags: ["cloud", "sdk", "viam-server", "networking", "apis", "ml model", "ml"]
+---
+
+The ML training API allows you to get data from and cancel ML training jobs taking place on the [Viam app](https://app.viam.com).
```

Comment:
Same suggestion as above. I feel like get data from makes me think that we can get the training data somehow from API

Reply:
Makes sense!